packages/plugins/experimental/plugin-transcription/src/components/TranscriptContainer.tsx

```diff
@@ -129,12 +129,32 @@ const summarizeTranscript = async (
 
 // TODO(dmaretskyi): Add example to set consistent structure for the summary.
 const SUMMARIZE_PROMPT = `
-  Create a summary of the transcript provided.
-  Reference specific people by name when possible.
-  Format the summary as a list of key points and takeaways.
-  Include action items in the summary.
-  Use markdown formatting for headings and bullet points.
-  Format the summary as a markdown document without extra comments like "Here is the summary of the transcript:".
+  <GOAL>
+  Create a markdown summary of the transcript provided.
```

Comment:
we've not been using xml in the prompts (only in the responses), so might be better to just keep it as markdown with level-2 headings.